docs/content/latest/integrations/smart-driver.md

```diff
@@ -0,0 +1,174 @@
+---
+title: JDBC Smart Driver
+linkTitle: JDBC Smart Driver
+description: Smart JDBC driver for YSQL
+aliases:
+menu:
+  latest:
+    identifier: smart-driver
+    parent: integrations
+    weight: 577
+isTocNested: true
+showAsideToc: true
+---
+
+JDBC Smart Driver for YSQL is a distributed driver based on the [PostgreSQL JDBC Driver](https://github.com/pgjdbc/pgjdbc).
+
+The following are the key features of JDBC Smart Driver:
+
+- It is **cluster-aware**, which eliminates the need for an external load balancer.
+
+  The driver package includes a `YBClusterAwareDataSource` class that uses one initial contact point for the YugabyteDB cluster as a means of discovering all the nodes and automatically gathering information about the state of each node (added, started, stopped, removed).
+
+- It is **topology-aware**, wich is essential for geographically-distributed applications.
+
+  The driver uses servers that are part of a set of geo-locations specified by topology keys.
+
+## Creating the Smart Driver
+
+You have a choice of obtaining the Smart Driver from Maven or creating it yourself.
+
+### How to Obtain the Driver from Maven
+
+To get the driver from Maven, add the following lines to your Maven project:
+
+```properties
+<dependency>
+  <groupId>com.yugabyte</groupId>
+  <artifactId>jdbc-yugabytedb</artifactId>
+  <version>42.2.7-yb-5</version>
+</dependency>
+```
+
+### How to Build the Driver
+
+To build the driver locally, follow this procedure:
+
+1. Clone the following repository:
+
+   ```sh
+   git clone https://github.com/yugabyte/jdbc-yugabytedb.git && cd jdbc-yugabytedb
+   ```
+
+2. Build and install into your local Maven folder by executing the following command:
+
+   ```sh
+   mvn clean install -DskipTests
+   ```
+
+3. Add the following lines to your Maven project:
+
+   ```properties
+   <dependency>
+       <groupId>com.yugabyte</groupId>
+       <artifactId>jdbc-yugabytedb</artifactId>
+       <version>42.2.7-yb-5-SNAPSHOT</version>
+   </dependency> 
+   ```
+
+## Using the Smart Driver
+
+Load balancing is performed using the following connection properties:
```

Comment:
We need to segregate the cluster aware load balancing with the topology aware.
For cluster aware only 'load-balance=true' is required

and for topology aware in addition to the above property even the second property is required which is 'topology-keys'

Reply:
Fixed